.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: Install npm depdencies
+    init: |
+      npm i -g npm
+      npm install
+    command: |
+      npm i -g npm
+      docker-compose up
```

Comment:
As Pauline said until theres a workaround the user needs to start it manually once the docker spin up finished.

Reply:
ok, cool, we can add it to the docs in bold.